Metadata/AudioMetadata.mm

```diff
@@ -47,6 +47,24 @@
 
 #include <wavpack/wavpack.h>
 
+using namespace APE;
+
+static TagLib::ID3v2::FrameListMap frameListMap(const TagLib::Tag* tag)
```

Comment:
This and `getFrameList` should probably be moved somewhere else, and not duplicated.